src/main/resources/db/migration/liquibase/changelog/db.changelog-master-1.0.2.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+  http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+
+  <property name="version" value="1.0.2" global="false"/>
+
+  <include file="./1.0.2/db.changelog-202504220000.sql"/>
```

Comment:
please, include `1.0.1` sql code too

Reply:
Added in both v1.0.1's changelog and v1.0.2's changelog